samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/SecureDeliverySeb.java

```diff
@@ -414,6 +405,16 @@ private String hashString(String string) {
         return HashingUtil.hashString(string);
     }
 
+    private boolean isEnabled(String siteId) {
+        switch(sebEnabled) {
+            case SEB_ALWAYS_ENABLED:
+                return true;
+            default:
+            case SEB_SITE_ENABLED:
+                return isSiteSebEnabled(siteId);
+        }
+    }
+
```

Comment:
return SEB_ALWAYS_ENABLED.equals(sebEnabled) ? true : isSiteSebEnabled(siteId);

This is shorter, but I understand this can have more cases in the future.